app/classes/document_query.rb

```diff
@@ -7,12 +7,16 @@ class DocumentQuery
     post_tags: ["\ue001"]
   }
 
+  attr_reader :language, :site_filters
+  attr_accessor :query
+
   def initialize(options)
     @options = options
+    @language = options[:language]
     if options[:query]
       site_params_parser = QueryParser.new(options[:query])
       @site_filters = site_params_parser.site_filters
-      @options[:query] = site_params_parser.remaining_query
+      @query = site_params_parser.stripped_query
```

Comment:
I changed `remaining_query` to `stripped_query` a) for consistency with usasearch and b) because `remaining_query` suggests that further parsing might occur (to me, anyway), while `stripped` suggests the final, fully parsed result.